Dockerfiles have go build error

```console
# github.com/GoogleCloudPlatform/grpc-gke-nlb-tutorial/echo-grpc/api
api/echo.pb.go:240:11: undefined: grpc.SupportPackageIsVersion6
api/echo.pb.go:250:5: undefined: grpc.ClientConnInterface
The command '/bin/sh -c apk add --no-cache git protobuf &&     go get github.com/golang/protobuf/protoc-gen-go &&     protoc --proto_path=api --go_out=plugins=grpc:api api/*.proto &&     GOPATH="/go" GO111MODULE=on CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go install -installsuffix "static" &&     echo 'nobody:x:65534:' > /src/group.nobody &&     echo 'nobody:x:65534:65534::/:' > /src/passwd.nobody &&     GRPC_HEALTH_PROBE_VERSION=v0.2.2 &&     wget -q -O /bin/grpc_health_probe https://github.com/grpc-ecosystem/grpc-health-probe/releases/download/${GRPC_HEALTH_PROBE_VERSION}/grpc_health_probe-linux-amd64 &&     chmod +x /bin/grpc_health_probe' returned a non-zero code: 2
```

I'm also facing the same issue.
```
# github.com/GoogleCloudPlatform/grpc-gke-nlb-tutorial/echo-grpc/api
api/echo.pb.go:240:11: undefined: grpc.SupportPackageIsVersion6
api/echo.pb.go:250:5: undefined: grpc.ClientConnInterface
The command '/bin/sh -c apk add --no-cache git protobuf &&     go get github.com/golang/protobuf/protoc-gen-go &&     protoc --proto_path=api --go_out=plugins=grpc:api api/*.proto &&     GOPATH="/go" GO111MODULE=on CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go install -installsuffix "static" &&     echo 'nobody:x:65534:' > /src/group.nobody &&     echo 'nobody:x:65534:65534::/:' > /src/passwd.nobody &&     GRPC_HEALTH_PROBE_VERSION=v0.2.2 &&     wget -q -O /bin/grpc_health_probe https://github.com/grpc-ecosystem/grpc-health-probe/releases/download/${GRPC_HEALTH_PROBE_VERSION}/grpc_health_probe-linux-amd64 &&     chmod +x /bin/grpc_health_probe' returned a non-zero code: 2
ERROR
ERROR: build step 0 "gcr.io/cloud-builders/docker" failed: step exited with non-zero status: 2
```

--- a/echo-grpc/go.mod
+++ b/echo-grpc/go.mod
@@ -21,5 +21,5 @@ go 1.12
 require (
 	github.com/golang/protobuf v1.3.1 // indirect
 	golang.org/x/net v0.0.0-20190522155817-f3200d17e092 // indirect
-	google.golang.org/grpc v1.21.0
+	google.golang.org/grpc v1.32.0
 )

--- a/reverse-grpc/go.mod
+++ b/reverse-grpc/go.mod
@@ -21,5 +21,5 @@ go 1.12
 require (
 	github.com/golang/protobuf v1.3.1 // indirect
 	golang.org/x/net v0.0.0-20190522155817-f3200d17e092 // indirect
-	google.golang.org/grpc v1.21.0
+	google.golang.org/grpc v1.32.0
 )

